### PR TITLE
updating DApp capitalization to standard use

### PR DESCRIPTION
**Pull request by roybotbot**
DApp -> dApp

### CODE REVIEW

File: docs/guides/walletconnect/walletconnect-user-guide.mdx [hunk 1]
```diff
@@ -130,30 +130,30 @@ If you would like to connect your Chia reference wallet to a different DApp, the
   warning ..\..\..\..\package.json: No license field
   $ next start
   ready - started server on 0.0.0.0:3000, url: http://localhost:3000
-  info  - Loaded env from C:\Users\User\Documents\GitHub\chia\chia-wallet-connect-dapp-test\.env.local
+  info  - Loaded env from C:\Users\User\Documents\GitHub\chia\chia-wallet-connect-dApp-test\.env.local
```
Comment (hoffmang9) [inline docs/guides/walletconnect/walletconnect-user-guide.mdx:133]: I was amazed that `git` handles this correctly. On MacOS you'll get a `chia-wallet-connect-dApp-test` directory regardless of the case in GitHub

**Review by hoffmang9 — CHANGES_REQUESTED** [review]
Good after the 404'd images issue is managed

**Comment by Rigidity** [comment]
@roybotbot are you able to fix conflicts & remove the image file path rename that 404's so we can get this merged

**Review by danieljperry — APPROVED** [review]
lgtm

**Review by danieljperry — APPROVED** [review]

**Review by hoffmang9 — APPROVED** [review]
Lgtm

**Pull request merged**